forge-gui/res/cardsfolder/e/endbringer.txt

```diff
@@ -2,7 +2,7 @@ Name:Endbringer
 ManaCost:5 C
 Types:Creature Eldrazi
 PT:5/5
-K:CARDNAME untaps during each other player's untap step.
+K:Untap CARDNAME during each other player's untap step.
```

Comment:
you can't just mess with a KW...

Reply:
Oops. Didn't do my due diligence checking that one. I'm closing this and making a new patch since it's outdated now.